hack/gen-test-target.sh

```diff
@@ -12,7 +12,7 @@ kebab-case-2-PascalCase() {
   local a=$1 b='' array
   IFS='-' read -r -a array <<< "$a"
   for element in "${array[@]}"; do
-    part="${element}"
+    part="${element//./_}"
```

Comment:
What is this change for?

Reply:
Simply deals with directory names which are like these : components/v1.0.0/....
The generated go code should not contains `.` in the function names. The code change replaces it with _